api/pom.xml

```diff
@@ -28,8 +28,8 @@
 
     <dependencies>
         <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
```

Comment:
Should we mark all the dependencies as provided in the pom.xml for API and TCKs in this PR? Unless, we want another PR to address that for Issue #220

Reply:
I'll do that in different issue, so we have issue resolved with separate PR for consistency.